content/docs/fragments.md

```diff
@@ -1,10 +1,10 @@
 ---
 id: fragments
-title: Fragments
+title: Fragmentler
 permalink: docs/fragments.html
 ---
 
-A common pattern in React is for a component to return multiple elements. Fragments let you group a list of children without adding extra nodes to the DOM.
+React'teki ortak şablon, bir bileşenin birden fazla öğe döndürmesidir. Fragmentler, Dom'a ekstra düğüm eklemeden bir çocuk listesini gruplandırmanıza izin verir.
```

Comment:
We decided to translate `child` as `alt eleman` (and `parent` as `ust eleman`) . Please see the `TRANSLATION.md` file. So:

> list of children -> alt elemanlar listesi